core/src/test/java/org/elasticsearch/index/store/ExceptionRetryIT.java

```diff
@@ -128,7 +128,7 @@ protected void sendRequest(Connection connection, long requestId, String action,
         refresh();
         SearchResponse searchResponse = client().prepareSearch("index").setSize(numDocs * 2).addStoredField("_id").get();
 
-        Set<String> uniqueIds = new HashSet();
+        Set<String> uniqueIds = new HashSete<>();
```

Comment:
It looks like you have a typo here.

It's very helpful it you can get gradle to pass  before raising a PR, so that these sorts of things get picked up by tooling rather than people needing to look at them.